sql/core/src/test/scala/org/apache/spark/sql/sources/DataSourceAnalysisSuite.scala

```diff
@@ -29,16 +29,24 @@ import org.apache.spark.sql.types.{DataType, IntegerType, StructType}
 
 class DataSourceAnalysisSuite extends SparkFunSuite with BeforeAndAfterAll {
 
+  protected override val doThreadAuditInSparkFunSuite = false
+
   private var targetAttributes: Seq[Attribute] = _
   private var targetPartitionSchema: StructType = _
 
   override def beforeAll(): Unit = {
+    doThreadPreAudit()
```

Comment:
Same thing here. This should be calling `super.beforeAll()`.

Reply:
Fixed.